test/specs/gfm/commonmark.0.29.json

```diff
@@ -2850,7 +2850,7 @@
   },
   {
     "markdown": "a*\"foo\"*\n",
-    "html": "<p>a*&quot;foo&quot;*</p>\n",
+    "html": "<p>a<em>&quot;foo&quot;</em></p>\n",
```

Comment:
according to the [commonmark spec](https://spec.commonmark.org/0.29/#example-352) that should not change.

Reply:
These common mark tests are derived from the spec so they should not be changed unless a new version of the spec comes out.

If a spec is marked with `shouldFail: true` that means it is failing and needs to be fixed in marked.

Reply:
marked as shouldFail